opm/output/eclipse/Summary.cpp

```diff
@@ -801,6 +819,13 @@ void Summary::add_timestep( int report_step,
     this->prev_time_elapsed = secs_elapsed;
 }
 
+void Summary::set_initial( const data::Solution& sol ) {
```

Comment:
That's my plan if/when that shows up, as this solution obviously doesn't scale (but is delightfully simple).